src/integration/tests/firstTest.test.jsx

```diff
@@ -98,45 +98,62 @@ describe("Test project in the app task list.", () => {
     });
   });
 
+  it("The user should not see the text 'Sed ut perspiciatis unde...' in the details of a task with an empty title.", () => {
+    render(<App />);
+    const taskMsg = "";
+    const input = screen.getByTestId("INPUT_TASK");
+    userEvent.clear(input);
+    userEvent.type(input, taskMsg);
+    const addTask = screen.getByTestId("ADD_BUTTON");
+    userEvent.click(addTask);
+    const task = within(screen.getByTestId(`TASK_CONTAINER_${taskMsg}`));
+    const infoTask = task.getByTestId("INFO_TASK_BUTTON");
+    userEvent.click(infoTask);
+
+    const details = screen.getByTestId("TASK_TITLE");
+    expect(details).toHaveTextContent(
```

Comment:
Shouldn't it be .not.toHaveTextContent?

```suggestion
    expect(details).not.toHaveTextContent(
```